hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/records/TxnAwareRecordsHistorian.java

```diff
@@ -65,10 +71,14 @@ public class TxnAwareRecordsHistorian implements RecordsHistorian {
 
     @Inject
     public TxnAwareRecordsHistorian(
-            RecordCache recordCache, TransactionContext txnCtx, ConsensusTimeTracker consensusTimeTracker) {
+            RecordCache recordCache,
+            TransactionContext txnCtx,
+            ConsensusTimeTracker consensusTimeTracker,
+            final @HandleThrottle FunctionalityThrottling handleThrottling) {
```

Comment:
Unfortunate that annotation `HandleThrottle` has no javadoc ...

(Also do annotations go before or after the `final`?  We usually write `@NonNull final SomeType argname` for example.)

Reply:
Re-ordered.